GitVersionExe/Program.cs

```diff
@@ -124,9 +125,21 @@ static int Run()
                             {
                                 throw new WarningException(string.Format("'{0}' variable does not exist", arguments.ShowVariable));
                             }
-                            Console.WriteLine(part);
+                            jsonOutput =part;
                             break;
                     }
+                    if (string.IsNullOrWhiteSpace(arguments.JsonOutputFile))
+                    {
+                        Console.WriteLine(jsonOutput);
+                    }
+                    else
+                    {
+                        var outfile = new FileInfo(arguments.JsonOutputFile);
```

Comment:
Do you mean the FileInfo, or GitVersion itself? I'm likely missing the obvious, but what call should write to the working directory. Out of interest, writing to the working directory is not adequate because it can be called from different locations (thanks TFS!)